files/mods.json

```diff
@@ -371,7 +366,7 @@
         "creator": "Sk1er LLC",
         "discordcode": "sk1er",
         "command": "/patcher",
-        "description": "Patcher optimizes the game in every possible way, fixes 70+ vanilla bugs along with some QOL features while maintaining both compatibility, and has almost no tradeoffs.",
+        "description": "Patcher is the ultimate Minecraft mod for framerate and performance, fixing over 70 issues within the games code and increasing your performance as a result.",
```

Comment:
I think a mention of the QOL features is important. It's not just a performance and bug fix mod that you throw in your mods folder and never touch again, it should be fine-tuned to the user's preferences.

Reply:
ehhhhhh that’s right but I don’t really know how to write that in

Reply:
If all else fails, just add a "and comes with a multitude of QOL/smaller features to better customize the game to your liking" or something along those lines to the end. I also don't really know a great way to make it flow well and be short, so I don't have much more to say on it

Reply:
Aight good enough for me I’ll patch something like that in later